src/components/page/PrTable.js

```diff
@@ -39,11 +58,57 @@ const PrTable = ({ query }) => {
     {},
     [query]
   );
+  const [prRecords, setPrRecords] = useState({});
+
+  useEffect(() => {
+    let mounted = true;
+    if (!data) return;
+
+    const refNums = [];
+    const prsByRefNum = {};
+
+    for (let pr of data) {
+      const refNum = pr.title.split(" ")[0].toUpperCase();
+      if (!/[A-Z]+-[0-9]+/.test(refNum)) continue;
+
+      refNums.push(refNum);
```

Comment:
This adds the entire string, not just the matching part of the reference. In my test PR, my title was `DEMO-26: Test`, which means it searched for `DEMO-26:` instead of `DEMO-26`.

Reply:
Fixed